its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -620,17 +620,18 @@ void testEsprojVueWithBackend() throws IOException {
 
     List<Issue> issues = TestUtils.allIssues(ORCHESTRATOR);
     List<String> ruleKeys = issues.stream().map(Issue::getRule).collect(Collectors.toList());
-    assertThat(ruleKeys).hasSize(5);
-    assertThat(ruleKeys).containsExactlyInAnyOrder(
+    assertThat(ruleKeys).hasSize(83);
+    assertThat(ruleKeys).contains(
```

Comment:
Educational: can this lead to flaky behavior? Does the order matter when using `.contains`?

Reply:
No, for order you need `containsExactlyInOrder`.
This is not even an exclusive list, I just added a single `css` issue so that we can avoid flaky behavior by adding all of them.